src/main/main.ts

```diff
@@ -443,10 +448,6 @@ app
 
 export {
   mainWindow,
-  recorder
+  recorder,
+  AppStatus
```

Comment:
Weird to export AppStatus here? It's imported from types.ts

Reply:
Sorry, I initially had it here and moved it to `types.ts` - I'll axe it.